spec/integration/lib/auth0/api/v2/api_stats_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'spec_helper'
+describe Auth0::Api::V2::Stats do
+  let(:client) { Auth0Client.new(v2_creds) }
+
+  describe '.active_users' do
+    let!(:active_users) { client.active_users }
+    it { expect(Integer(active_users)).to be >= 0 }
+  end
+
+  describe '.daily_stats' do
+    let(:from) { Time.zone.today.prev_day.strftime('%Y%m%d') }
+    let(:to) { Time.zone.today.strftime('%Y%m%d') }
+    let(:daily_stats) { client.daily_stats(from, to) }
+    it { expect(daily_stats.size).to be > 0 }
```

Comment:
I see no reason why this should be greater than 0 but for previous tests ran. Should we add a step login a user in?